### ISSUE TITLE
GeoClimate data for UMEP use

### ISSUE THREAD

**Issue by j3r3m1** [orbisgis/geoclimate#865]
It seems calling GeoClimate from Python is very simple using subprocess. When we call GeoClimate for a given territory using OSM data, GeoClimate:
1. Download the data and format it into GIS tables (vector data): buildings, roads, water, impervious, etc.
2. Estimate building height for buildings having no info in OSM
3. Calculate urban canopy parameters and LCZ at 2 different scales:
    - Topographical Spatial Units: it is a continuous and homogeneous way to divide the space using topographic constraints: road and railway center lines, vegetation, impervious and water areas, administrative boundarie)
    - Regular rectangular grid scale

I suppose that most of the GIS layers might be useful in the UMEP tool and maybe also the LCZ data generated by GeoClimate. @biglimp ?

If to, to make it possible to download all data except the LCZ, we need to enable the use of the GeoClimate workflow when indicatorUse is empty. Currently, [the building height estimation is activated even though indicatorUse is empty](https://github.com/orbisgis/geoclimate/blob/a811f3321e44b728ff30e8d415eb4366da06c91c/geoindicators/src/main/groovy/org/orbisgis/geoclimate/geoindicators/WorkflowGeoIndicators.groovy#L1255) but however this part of the code is within the rsu_indicator workflow which is only applied [if the indicatorUse is not empty](https://github.com/orbisgis/geoclimate/blob/a811f3321e44b728ff30e8d415eb4366da06c91c/osm/src/main/groovy/org/orbisgis/geoclimate/osm/WorkflowOSM.groovy#L526).

If we activate it, we should just make sure that the rest of the code within the rsu indicator calculation is performed only if indicatorUse is not empty (which is currently not the case).

**Comment by biglimp** [comment]
Yes!. The more opportunities to retrieve spatial data for urban climate modelling, the better. We have some simple tools in UMEP to derive ground and building digital surface models (https://umep-docs.readthedocs.io/en/latest/pre-processor/Spatial%20Data%20DSM%20Generator.html) but usually building heights are absent for many areas around the Globe.
